### PR TITLE
New Progressive Impact Development Website (with sketch files)

### PR DESCRIPTION
**Pull request by CodyJasonBennett**
### This site features a smooth three-page design, featuring a changelog page fed from a markdown file from [/src/docs/changelog.md](https://github.com/CodyJasonBennett/ImpactWebsite/blob/master/src/docs/changelog.md).

[Home](https://impactmc.netlify.com)
[Changelog](https://impactmc.netlify.com/changelog)
[Donate](https://impactmc.netlify.com/donate)

### All I ask is for you to preserve the footer rebounding to [my site](https://codyb.co/projects/impact) and I can say I helped design and build your website by displaying its screens on [my portfolio](https://codyb.co/project/impact).

[Home](https://codyb.co)
[Reveal](https://codyb.co/projects/impact)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "skipThirdPartyRequests": true,
+    "headless": false
+  },
+  "browserslist": [
+    ">1%",
+    "not dead",
+    "not ie 11",
+    "not op_mini all"
+  ]
+}
```
Comment (LeafHacker) [inline package.json:48]: After experimenting with a build/deploy workflow in #18 I've settled on using github-page's build in integration with [Jekyll](https://jekyllrb.com/) instead. Unfortunately I can't get behind using react for this.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <meta name="twitter:site" content="@ImpactDevelopment">
+    <meta name="twitter:image" content="%PUBLIC_URL%/social-image.png"}>
+	<meta name="author" content="Cody Bennett">
+
+    <title>Impact Development</title>
+  </head>
+  <body>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
```
Comment (LeafHacker) [inline public/index.html:34]: This is unacceptable. JS should improve the experience not be a minimum requirement.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                  </Transition>
+                </TransitionGroup>
+              </React.Fragment>
+            )} />
+          </Router>
+        </React.Fragment>
+      </ThemeProvider>
+    </HelmetProvider>
+  );
+};
```
Comment (LeafHacker) [inline src/App.js:72]: JSX is cool and awesome, however if we were to go down this route gh-pages wouldn't be the place to do it. IMO react should always be rendered server-side and simply add to the experience client-side. That'd likely require paying for a service such as heroku to host the site (depending how much traffic we get @leijurv ?).

Seems overkill for what only needs to be a simple website with a download link.

**Review by LeafHacker — CHANGES_REQUESTED** [review]
Thank you for your work designing and building this webapp. I have to say it looks very clean and modern, and the site could do with a refresh soon.

However as much as it pains me, I can't accept this PR. Hopefully my inline comments help clarify my reasons.

I'd encourage you to always discuss your ideas with a project's team before implementing them, especially if you have to dedicate much time or effort to that implementation. While it may seem long winded, it will save you time in the long run and avoid situations like this where you spend time creating something great, but that isn't the something the project actually wants.

Thank you once again for your effort.

File: public/index.html (hunk 2 above)
Comment (ZeroMemes) [reply public/index.html:34]: The ad service requires JS, and so does the handling of operating system detection in the website currently.

File: src/App.js (hunk 3 above)
Comment (CodyJasonBennett) [reply src/App.js:72]: This was merely an exercise that I wanted to share to get feedback.
However, if you are looking for help, I can work with your team to design a website from the ground-up as well as build it. It doesn't have to be built with React; I can make static websites all-the-same. I've done this previously with other sites like [MystGang's portfolio](https://codyb.co/projects/mystgang/). I am looking to sharpen my skills without any cost to your team. I found Impact Development through your lead developer in previous projects.

Anyways, I have a server you can use if hosting is an issue.
Alternatively, there are services like [Netlify](https://www.netlify.com/) which are free to use.
Netlify, in particular, builds from your stable branch, keeping your site up-to-date.
You can work with these tools from your browser with [CodeSandbox](https://codesandbox.io/), which will take care of Github and deployment.

With my server, I can also hook you up with my Cloudflare subscription which makes the site much faster and still gives you access to SSL. For reference, all of my works have scored in the top 95% of websites in speed and accessibility thanks to this CDN.

**Pull request closed without merging**